.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "build/storage/spdk"]
```

Comment:
It feels like we should update a README somewhere to make sure we note you now to clone the repository with `--recursive` here, but I see our READMEs are a bit of a mess. I filed #156 to track this.

Reply:
Yeah, do we want to use submodules or do we keep using git clone/downloads in scripts? I prefer the latter but possibly I don't know the positive points of using submodules...

Reply:
There are opponents and proponents of both approaches. We had the guideline to include SPDK as a submodule with IPDK.

Reply:
Lets leave the submodule in here for now to move this forward. I believe as we look to figure out a better long-term strategy for the repositories which make up IPDK, we can figure out if the submodule is necessary or not.